tests/jax/components/training/losses_test.py

```diff
@@ -0,0 +1,191 @@
+from types import SimpleNamespace
+from typing import Callable, Tuple
+
+import jax
+import jax.numpy as jnp
+import pytest
+
+from mava.components.jax.training.losses import (
+    MAPGTrustRegionClippingLossConfig,
+    MAPGWithTrustRegionClippingLoss,
+)
+from mava.systems.jax.trainer import Trainer
+
+
+class Mock_net:
+    """Creates a mock network for loss function"""
+
+    def apply(
+        parameters: jnp.array, observation: jnp.array
+    ) -> Tuple[jnp.ndarray, jnp.ndarray]:
+        """Mock function to apply the network to training data"""
+        return jnp.array(observation), jnp.array(observation)
+
+
+def log_prob(distribution_params: jnp.array, actions: jnp.array) -> jnp.ndarray:
+    """Mock function to return fixed log probs"""
+    return distribution_params + actions
```

Comment:
Can we not just concatenate here as well?

Reply:
Since they're both arrays I added them together to pretend we're using them to calculate the log_probs. If they're concatenated it would change the output size.